folder_structure.md

```diff
@@ -254,6 +255,28 @@ The files are all produced by the [profiling-recipe](https://github.com/jump-cel
 
 - `2021_04_26_Batch1` is the batch and `BR00117035` is the plate
 
+### `embeddings` folder structure:
+
+The structure here is very similar to `analysis`
+
+```
+└── embeddings
+    ├── 2021_04_26_Batch1
+    │   ├── BR00117035
+    │   │   └── efficientnet_v2_imagenet1k_s_feature_vector_2_ec756ff
+    │   │       ├── BR00117035-A01-1
```

Comment:
Is not it redundant to have plate id here, or it is implied that someone might want to download site embedding separately?

Reply:
Here, too, I am following what we have used in the past with CellProfiler, but indeed, we can skip the plateid prefix here. I will make that change